### ISSUE TITLE
How to find values for stride in nn.ConvTranspose2d

### ISSUE THREAD

**Issue by StatMixedML** [StatMixedML/ConvTS-Mixer#1]
# Description

For our Mixer models, we need to map the output of the mixer blocks from the patch-dimension back to the original-dimension. 

# Details

Suppose, the output of the mixer-block is of shape `torch.Size([128, 32, 12, 4])`. Using the nn.ConvTranspose2d, we need to map it to `torch.Size([128, 32, 24, 137])`, where the dimensions are as usual `[B C H W]`.

So if we start with the following (https://pytorch.org/docs/stable/generated/torch.nn.ConvTranspose2d.html)

```python
H_out=(H_in−1)*stride_H−2*padding[0]+dilation[0]*(kernel_size_H−1)+output_padding[0]+1
W_out=(W_in−1)*stride_W−2*padding[1]+dilation[1]*(kernel_size_W−1)+output_padding[1]+1
```

and set `padding=0, output_padding=0` and assume that kernel_size=(11,11) is known, we get

```python
H_out = (H_in - 1) * stride_H + kernel_size_H
W_out = (W_in - 1) * stride_W + kernel_size_W
```

Solving for the stride gives
```python
stride_H = (H_out - kernel_size_H) // (H_in - 1)
stride_W = (W_out - kernel_size_W) // (W_in - 1)
```

Using these values in the nn.ConvTranspose2d does not give the correct output dimensions. Instead of returning a tensor of shape `torch.Size([128, 32, 24, 137])` it gives `torch.Size([128, 32, 22, 137])` . Not sure where the problem is. 

@kashif @pablovicente Any ideas?
